currency_converter.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import sys
+
+from forex_python.converter import CurrencyCodes
+from forex_python.converter import CurrencyRates
+from forex_python.converter import RatesNotAvailableError
```

Comment:
There is `README.md` with requirements section.
But I guess `requirements.txt` file is a python thing, which I didn't know.
It's just about getting used to another language with different set of tools. This transition can take a few days.